app/models/blog.py

```diff
@@ -12,15 +12,13 @@ class Blog(Base):
     __tablename__ = "blogs"
 
     id = Column(Integer, primary_key=True, index=True)
-    title = Column(String, index=True)
-    content = Column(String)
+    title = Column(String(2000), nullable=False, default="No title")
```

Comment:
i title can be 2000 character long?

Reply:
Yes blog title can reach 2000 char in some cases